specification/common-types/resource-management/v2/types.json

```diff
@@ -179,6 +186,26 @@
       },
       "x-ms-azure-resource": true
     },
+    "ExtendedLocation": {
+      "description": "The resource model definition representing extended location",
+      "properties": {
+        "type": {
```

Comment:
what is the mutability of this property?  Can it be changed on update, or only on create?  Please appropriately mark with x-ms-mutability.  This applies to 'name' as well.